### PR TITLE
fix(onechunk): Small code tidying, speed clar. and disclaimer

### PR DESCRIPTION
**Pull request by Hyenadae**
Boysanic mentioned something, so I gave the code a quick look over and made some small changes

### CODE REVIEW

File: filter9000-boinc.cpp [hunk 1]
```diff
@@ -289,15 +296,16 @@ int main(int argc, char **argv) {
...
     }
+
     #ifdef BOINC
         boinc_begin_critical_section();
     #endif
     time_t elapsed = (time(NULL) - start) + elapsed_chkpoint;
     double done = (double)total;
-    double speed = done / (double) elapsed;
-    fprintf(stderr, "\nSpeed: %.2lf/s\n", speed );
+    double speed = (done / (double) elapsed) * 65535;
```
Comment (BoySanic) [inline filter9000-boinc.cpp:322]: I'm still not entirely sold on this.

The number looks bigger, but, it's not as easily comparable to the task size.

File: filter9000-boinc.cpp [hunk 2]
```diff
@@ -289,15 +296,16 @@ int main(int argc, char **argv) {
...
+    fprintf(stderr, "\nSpeed: %.2lf worldseeds/s\n", speed );
```
Comment (BoySanic) [inline filter9000-boinc.cpp]: Same comment as above

File: filter9000-boinc.cpp [hunk 3]
```diff
@@ -289,15 +296,16 @@ int main(int argc, char **argv) {
...
     fprintf(stderr, "Done.\n");
-    fprintf(stderr, "Processed: %llu seeds in %.2lfs seconds.\n", total, (double) elapsed_chkpoint + (double) elapsed );
+    fprintf(stderr, "Processed: %llu input seeds in %.2lfs seconds.\n", total, (double) elapsed_chkpoint + (double) elapsed );
```
Comment (BoySanic) [inline filter9000-boinc.cpp]: If we were processing input seeds and showing stats for input seeds, why show world seeds in the previous print?

Confusing to show stats for 65536 times the work when we only tell them they processed a number 65536x smaller.

**Review by BoySanic — CHANGES_REQUESTED** [review]

**Comment by Hyenadae** [comment]
I mean at least checkpointing seems to work!

**Review by BoySanic — APPROVED** [review]
LGTM, I hope :shipit:

**Pull request merged**